tests/Integration/Config/ConfigFileFinderTest.php

```diff
@@ -11,7 +11,7 @@ public function test_config_loader_throw_when_it_cant_find_a_psh_file()
     {
         $loader = new ConfigFileFinder();
 
-        $this->expectException(\RuntimeException::class);
+        $this->setExpectedException(\RuntimeException::class);
```

Comment:
It all depends on the distribution of humbug + phpunit you use. This project for example has a local phar file. The humbug autodiscovery still has room for improvement.